elasticdl_preprocessing/utils/analyzer_utils.py

```diff
@@ -0,0 +1,146 @@
+"""This util gets statistics from the environment and return a default value
+if there is no statistics in the environment. So user can set a default value
+to perform unit tests.
+"""
+import os
+
+
+def get_min(feature_name, default_value):
+    """Get the min value of numeric feature from the environment.
+    Return the default value if there is no the statistics in
+    the environment.
+
+    Args:
+        feature_name: String, feature name or column name in a table
+        default_value: Float.
+
+    Return:
+        Float
+    """
+    env_name = "_" + feature_name + "_min"
+    min_value = os.getenv(env_name, None)
+    if min_value is None:
+        return default_value
+    else:
+        return float(min_value)
+
+
+def get_max(feature_name, default_value):
+    """Get the max value of numeric feature from the environment.
+    Return the default value if there is no the statistics in
+    the environment.
+
+    Args:
+        feature_name: String, feature name or column name in a table
+        default_value: Float.
+
+    Return:
+        Float
+    """
+    env_name = "_" + feature_name + "_max"
+    max_value = os.getenv(env_name, None)
+    if max_value is None:
+        return default_value
+    else:
+        return float(max_value)
+
+
+def get_avg(feature_name, default_value):
```

Comment:
get_mean?

Reply:
"AVG" is the keyword in SQL but `mean` is not. So I think `get_avg` is more suitable.